playbooks/roles/install-wetty/tasks/wetty.yml

```diff
@@ -0,0 +1,266 @@
+---
+
+- name: Install required packages
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - make
+    - nmap-ncat
+    - npm
+    - gcc-c++
+  become: true
+
+- name: Check if directory exist example
+  stat:
+    path: /etc/letsencrypt/live/{{ openshift_public_hostname }}
+  register: cert_directory
+
+- name: Create directory for certs
+  file: path=/etc/letsencrypt/live/{{ openshift_public_hostname }} state=directory
+  become: true
+  when: not cert_directory.stat.exists
+
+- name: Get SSL cert for wetty
+  command: >-
+    docker run --rm --name certbot
+    -v "/etc/letsencrypt:/etc/letsencrypt:z"
+    -v "/var/lib/letsencrypt:/var/lib/letsencrypt:z"
+    -e AWS_ACCESS_KEY_ID={{ lookup('env', 'AWS_ACCESS_KEY_ID') }}
```

Comment:
We can document that these vals need to be environment vars in the provisioner readme